test/src/unit-filter-pipeline.cc

```diff
@@ -3319,8 +3319,8 @@ TEST_CASE("Filter: Test positive-delta encoding", "[filter][positive-delta]") {
       CHECK(tile.write(&val, i * sizeof(uint64_t), sizeof(uint64_t)).ok());
     }
 
-    CHECK(
-        !pipeline.run_forward(&test::g_helper_stats, &tile, nullptr, &tp).ok());
+    CHECK_THROWS(
```

Comment:
I have found a few times our tests don't really validate what we think they are validating... Could we switch these to CHECK_THROWS_WITH or something similar?

Reply:
Done